sentry-android-core/src/main/java/io/sentry/android/core/internal/gestures/SentryGestureListener.java

```diff
@@ -184,6 +184,7 @@ private void addBreadcrumb(
         hint);
   }
 
+  // TODO should we start a new trace here if performance is disabled?
```

Comment:
Need help deciding whether this should create a new `PropagationContext` with random `traceId` and `spanId` or if we rather want to keep the previous trace going.

Reply:
OK, can start a new trace here too.